File: C/1044.c
#include <stdio.h>

int main(){
    int x,y;
    scanf("%d%d",&x,&y);
    if(x > y){
        if(x%y == 0){
            printf("Sao Multiplos\n");
            return 0;
        }
        else{
            printf("Nao sao Multiplos\n");
            return 0;
        }
    }
    else{
        if(y%x == 0){
            printf("Sao Multiplos\n");
            return 0;
        }
        else{
            printf("Nao sao Multiplos\n");
            return 0;
        }
    }
}

File: python/1044.py
lista = list(map(int, input().split()))

if(lista[0] > lista[1]):
    if(lista[0] % lista[1] == 0):
        print("Sao Multiplos")
    else:
        print("Nao sao Multiplos")
else:
    if(lista[1] % lista[0] == 0):
        print("Sao Multiplos")
    else:
        print("Nao sao Multiplos")
